wallet/src/ui/app/pages/transfer-nft/TransferNFTForm.tsx

```diff
@@ -52,29 +50,8 @@ function TransferNFTForm({
             </div>
 
             <div className={st.group}>
-                <label className={st.label}>Amount:</label>
-                <Field
-                    component={NumberInput}
-                    allowNegative={false}
-                    name="amount"
-                    value={transferCost}
-                    className={st.input}
-                    disabled={true}
-                />
-                <div className={st.muted}>
-                    Available balance:{' '}
-                    {intl.formatNumber(
-                        BigInt(gasBalance),
-                        balanceFormatOptions
-                    )}{' '}
-                </div>
-                <ErrorMessage
-                    className={st.error}
-                    name="amount"
-                    component="div"
-                />
+                * Total transaction fee estimate (gas cost): 430 {GAS_SYMBOL}
```

Comment:
use `transferCost`

Reply:
I think we are missing one here. We should export `transferCost`(nit: we should rename it to `DEFAULT_NFT_TRANSFER_GAS_FEE`) if it is defined in another file